CHANGELOG.md

```diff
@@ -8,11 +8,19 @@ _**For better traceability add the corresponding GitHub issue number in each cha
 
 ## [Unreleased]
 
-## [5.2.0] - 2024-07-03
+### Added
 
 ### Fixed
 
 - Access and Usage Policy Validation flow correction. #757
+
+### Changed
+
+
+## [5.2.0] - 2024-07-03
+
+### Fixed
```

Comment:
Why was this moved to unreleased? The fix is already released as 5.2.0 and this PR only removes a unused method